solr/solrj/src/java/org/apache/solr/client/solrj/impl/LBSolrClient.java

```diff
@@ -612,6 +603,7 @@ private void startAliveCheckExecutor() {
     if (aliveCheckExecutor == null) {
       synchronized (this) {
         if (aliveCheckExecutor == null) {
+          log.debug("Starting aliveCheckExecutor");
```

Comment:
[0] Not sure if you intend to keep these debug lines, but if so, might be worth adding something to the message to indicate which client it belongs to, since apps might have multiple.